docs/channels_docs.md

```diff
@@ -0,0 +1,24 @@
+# Channels Documentation
```

Comment:
Can we add a link to this file in the docs readme also? See https://github.com/nicklaw5/helix/blob/master/docs/README.md

Reply:
Done! Thanks for the quick replies :D